lib/config/presets/index.ts

```diff
@@ -138,6 +142,15 @@ export function parsePreset(input: string): ParsedPreset {
       .map((elem) => elem.trim());
     str = str.slice(0, str.indexOf('('));
   }
+  if (presetSource === 'http') {
+    [repo] = str.split('#', 1);
+    if (repo === str) {
+      presetName = undefined;
+    } else {
+      presetName = str.substring(repo.length + 1);
```

Comment:
@rarkins I think we like to deprecate sub presets? so we shouldn't support it in this new type?

Reply:
One preset per file only?

Reply:
I'm it sure if we'll ever be able to though